recipes/jtango/build.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -p ${PREFIX}/usr/share/java
+
+install -m 0644 JTango-${PKG_VERSION}.jar ${PREFIX}/usr/share/java
+ln -s JTango-${PKG_VERSION}.jar ${PREFIX}/usr/share/java/JTango.jar
```

Comment:
I was able to generate a list of third party notices in the git tree using the following:

```suggestion
ln -s JTango-${PKG_VERSION}.jar ${PREFIX}/usr/share/java/JTango.jar

mvn install
mvn license:aggregate-third-party-report
```
Then include `target/site/aggregate-third-party-report.html` as part of the `license` field.